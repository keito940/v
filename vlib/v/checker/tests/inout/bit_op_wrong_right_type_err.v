fn main() {
	1 | 0.5
}