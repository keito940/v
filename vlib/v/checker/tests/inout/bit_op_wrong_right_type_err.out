vlib/v/checker/tests/inout/bit_op_wrong_right_type_err.v:2:4: error: operator | not defined on right type `f64`
    1| fn main() {
    2|     1 | 0.5
             ^
    3| }
